docs/USAGE-GUIDE.md

```diff
@@ -332,7 +332,7 @@ supported status request type is OCSP, **S2N_STATUS_REQUEST_OCSP**.
 ```c
 typedef enum { S2N_CERT_AUTH_NONE, S2N_CERT_AUTH_REQUIRED, S2N_CERT_AUTH_OPTIONAL } s2n_cert_auth_type;
 ```
-**s2n_cert_auth_type** is used to declare what type of client certificiate authentication to use.
+**s2n_cert_auth_type** is used to declare what type of client certificate authentication to use.
```

Comment:
You clearly got your spellchecker working again!

Reply:
is used - change to used

Reply:
@Lucy-writer, "is used" is in a lot of places in this doc, we should probably remove them all at once rather than in this pr, if we do want this changed.